users/faq.md

```diff
@@ -1 +1,33 @@
 # FAQ
+## Related to usage in Planetariums
+1. How do I get a fulldome (fisheye) output from OpenSpace?
+
+When you first start up OpenSpace, you get a screen where you can choose a Profile and choose Window Options. Window Options is where you can choose to use a fisheye configuration. 
+
+2. How can I save screenshots in a resolution higher than my screen resolution? For example, my computer screen can only display 1920x1080, but I want to capture 4096x4096 pngs using the session recording function.
+
+You can choose to display a different resolution, and render at a different resolution. The single.json Window options configuration file is an example which has a display resolution of 1280x720 (the "size" parameter) but renders pngs at 2560x1440 (the "res" parameter). These configuration files are located in the 'config' directory - you can write your own for your desired resolution.
+
+3. How can I get rid of the text and GUI elements in my planetarium display?
+
+There are several ways to do this. For example, you could
+(a) press TAB and SHIFT+TAB to toggle visibility of the GUI and text elements respectively, or
+(b) create or use a Window options configuration file which renders the GUI and the data visualization separately on different windows, for example the single_fisheye_gui.json configuration file or
+(c) create a [html control page](https://docs.openspaceproject.com/en/latest/how-to/html-controls-setup/index.html) and use that to control the navigation, hiding the GUI.
+
+4. How do I ensure that all the data sets I need are loaded before playing back a recorded path? Can I play back without an internet connection?
+
+Go into the openspace.cfg file and set ModuleConfigurations->GlobeBrowsing->MRFCacheEnabled to true. Run OpenSpace again it will cache the globebrowsing data for planet/moon locations you visit. Then later, it should use the cached data for those same locations. There is also an Offline profile that you can select from the launcher, and see if that has the content you want.
+
+## Related to running OpenSpace on MacOS, Linux and unsupported hardware 
+1. Why do you only provide Intel Mac packages? Why not packages for Apple Silicon Macs?
+
+OpenSpace uses the OpenGL cross-platform library. Unfortunately, Apple does not yet support the latest versions of OpenGL on M1/M2/M3 Macs. Asahi Linux has successfully ported OpenGL to Apple Silicon Macs, so those with Apple Silicon Macs could, if they're feeling adventurous, [install Asahi Fedora remix](https://asahilinux.org/fedora/) and run OpenSpace on their Apple Silicon Macs.
```

Comment:
```suggestion
When starting an x86 or x64 application on MacOS, it automatically runs through the Rosetta 2 translation layer, so that the ARM chip in the machine can execute the instructions.  That part is not really a big deal and it would come with some performance loss that could be acceptable.  However, we are using OpenGL as our graphics pipeline, which Apple has deprecated on their platforms for a number of years now.  The way they still support it on M1 is a separate conversion layer that transforms the OpenGL calls into Metal 3 (the Apple-specific graphics API) before it gets executed by the GPU cores on the M1/M2 chip.  Apple decided for some very strange reason to not support double precision floating point operations (https://developer.apple.com/metal/Metal-Shading-Language-Specification.pdf Section 2.1) which we use everywhere in OpenSpace to get enough precision for the whole universe. So as soon as we run any OpenGL that uses double precision, it no longer uses the GPU cores built into the chip, but reverts to software-based rendering, which kills the performance entirely.

Asahi Linux has successfully ported OpenGL to Apple Silicon Macs, so those with Apple Silicon Macs could, if they're feeling adventurous, [install Asahi Fedora remix](https://asahilinux.org/fedora/) and run OpenSpace on their Apple Silicon Macs.
```

Reply:
Clarified this a bit as it is not the lack of modern OpenGL that is the issue, but the lack of support for double precision floating point numbers in the Metal API.

Are you sure that it works on Asahi? I was under the impression that the hardware wasn't supporting double.

Reply:
> Clarified this a bit as it is not the lack of modern OpenGL that is the issue, but the lack of support for double precision floating point numbers in the Metal API.
> 
> Are you sure that it works on Asahi? I was under the impression that the hardware wasn't supporting double.
I don't have a Mac with Apple Silicon myself, so I can't try it out myself. I had recently read this - https://arstechnica.com/gadgets/2024/02/asahi-linux-projects-opengl-support-on-apple-silicon-officially-surpasses-apples/ - there, Asahi's Alyssa says that they have successfully remapped OpenGL calls to Apple Silicon using some tricks. Perhaps something similar to https://github.com/philipturner/metal-float64